fitbenchmarking/fitbenchmarking
# IVP analytic jacobians

#### Description of Work

Fixes #830 

#### Testing Instructions

1. Run Fitbenchmarking with Data Assimilation problems and analytic jacobians, and check that results are comparable to using other jacobians
2.
3.

Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?

Style: Is the coding style consistent? Is anything overly confusing?

Documentation: Is there a suitable change to documentation for this change?
